README.md

```diff
@@ -25,32 +25,40 @@ buildscript {
 plugins {
   id "net.ltgt.apt" version "0.11"
   id "net.ltgt.errorprone" version "0.0.11"
-  id "java"
+  id "java-library" // Or just plain old "java"
 }
 
 dependencies {
     apt "com.uber.nullaway:nullaway:0.1.2"
 
-    compile "com.google.code.findbugs:jsr305:3.0.2"
+    // Optional, some source of nullability annotations
+    compileOnly "com.google.code.findbugs:jsr305:3.0.2"
 
     errorprone "com.google.errorprone:error_prone_core:2.1.1"
     errorprone "org.checkerframework:dataflow:2.1.14"
-
 }
 
-compileJava {
+tasks.withType(JavaCompile) {
```

Comment:
This applies the same checks to test code as well. We should mention that and provide a way for folks to only apply to main code (as it is now) as test code may not be as important for null warnings

Reply:
I can add a bit clarifying. I don't know off hand how this can be made to exclude test code though, do you?

Reply:
FWIW I think our default configuration should not run NullAway on test code; the payoff there is less clear.  The point of this change was to unify Java and Android projects, but this PR already has a small separate section for Android.  Couldn't we just make Android different in this respect as well?

Reply:
I think we should leave this line unchanged as that indicates we are just talking about main code